src/app/api/imageInput/route.ts

```diff
@@ -0,0 +1,49 @@
+import { ChatOpenAI } from "@langchain/openai";
+import { HumanMessage } from "@langchain/core/messages";
+import { env } from "@/app/env.mjs";
+import { NextResponse } from "next/server";
+
+const chat = new ChatOpenAI({
+  modelName: "gpt-4-vision-preview",
+  maxTokens: 1024,
+  openAIApiKey: env.OPEN_AI_API_KEY,
+});
+
+export async function POST(request: Request) {
+  let formData = await request.formData();
+  if (!formData || !formData.has("file")) {
+    return NextResponse.json(
+      { result: "no data received", success: false },
+      { status: 400 },
+    );
+  }
+
+  const file = formData.get("file");
+  const imageInput: any = formData.get("imageInput");
+  if (file && file instanceof File) {
```

Comment:
Here is `File`
```suggestion
  if (file && file instanceof File) {
```
Here it is

Reply:
Resolve it